source/Octopus.Tentacle.Contracts/Octopus.Tentacle.Contracts.csproj

```diff
@@ -28,7 +28,7 @@
     </Otherwise>
   </Choose>
   <ItemGroup>
-    <PackageReference Include="Halibut" Version="7.0.682" />
+    <PackageReference Include="Halibut" Version="8.0.913-pull-617" />
```

Comment:
Note: Will need to update this to actual Halibut release once that has been merged/built.